django_project/frontend/src/pages/Admin/Importer/Form/Extensions/SDMX/DsdComponents/config.js

```diff
@@ -0,0 +1,10 @@
+// config.js
+const API_URLS = {
```

Comment:
I think we need to make the domain an input
So user can use other sdmx api to handle this